### ISSUE TITLE
Generator works except on Windows 10 with Node 10

### ISSUE THREAD

**Issue by eddyystop** [yeoman/generator#1098]
I have a generator that works on Ubuntu, Mac, and Windows 10 with Node 8.

It also (reportedly) runs on Win10 with Node 10 when run on either mintty.exe or Git bash from the git package (https://git-scm.com/download/win).

However it hangs when run on Win10 with Node 10 on either PowerShell or command.exe.

The "prompting" method runs fine and the "this.prompt" promise resolves OK. The problem is the "writing" method" is never run.

You can see the problem by going to an empty folder and running
```txt
npm i -g @feathers-plus/cli
set DEBUG=gen*
feathers-plus generate app
```
and pressing Enter on all the prompts.

You can see https://github.com/feathers-plus/generator-feathers-plus/blob/master/generators/app/index.js#L173 is run, but not https://github.com/feathers-plus/generator-feathers-plus/blob/master/generators/app/index.js#L178 .

How can I proceed to find what's causing this? Thanks in advance.

**Comment by eddyystop** [comment]
Can ideas?

**Comment by eddyystop** [comment]
The problems been tracked down further. The generator correctly calls for multiple files to be created using template, JSON and copy. The hang seems to occur before the first file is created, so I'm assuming it hangs on the first request, which is to copy one file to the generated target.

**Comment by bisubus** [comment]
Is there a reason to consider that the problem is specific to Windows 10 and not Windows platform? The generator stalls with Windows 7 and Node 10 for me, while it runs with Node 8.

**Comment by eddyystop** [comment]
I was not aware of a problem with Windows 7 also. Its a bit surprising that it works between 7 and 10.

I did not have the time to trace the problem further. Luckily its solid.

**Comment by eddyystop** [comment]
Any news?

**Comment by eddyystop** [comment]
The problem occurs with Windows 10 and Node 10, but not with Node 8.

**Comment by mdewey** [comment]
Has there been any solution/workaround/movement? I am running into the same issue with a generator I am working on? 

I am about to dive into the `yo` code to see what I can do and was wondering if there was any more knowledge to absorb.

**Comment by Hobbs1210** [comment]
I believe the issue is related to Inquirer. I came across this [Windows 10 hang ](Https://github.com/SBoudrias/Inquirer.js/issues/767) issue.

**Comment by delebash** [comment]
This problem was reported on Nov 16 2018 yet there is still no fix.  Please someone from the maintainers group look into this.

**Comment by delebash** [comment]
Any updates?

**Comment by Clarence-L** [comment]
Generator hangs in windows cmd and powershell:
Windows 8.1
node 10.15.3
and
Windows 10.0
node 10.15.3

I was able to run the feathers-plus generator from an emacs shell, but was not able to change the multiple choice options on the auth generator since the interactive prompts were not responding to arrow keys and spacebar inputs (something to do with (n)curses I believe).

Was really hoping on using feathers-plus for this project.

**Comment by Clarence-L** [comment]
Found a workaround using cygwin and mintty:

See: [https://github.com/feathers-plus/generator-feathers-plus/issues/103#issuecomment-477351975](url)

**Comment by bisubus** [comment]
The problem is specific to Node 10+ and Windows in general. I published a set of Node executable wrappers to use specific Node version together with global Node in the same project, specifically `@ex-machine/node8`. It can be used to run entry script of a generator:

```
node8 ./node_modules/@feathers-plus/cli/lib/index.js
```

**Comment by soufianesakhi** [comment]
I have tried to debug this issue, it seems to hang for me when trying to log the message "Your generator must be inside a folder named generator-xxx\nI'll automatically create this folder.", exactly at `console.error.apply(console, arguments)` (yeoman-environment/lib/util/log.js:80)

When launching `yo` without arguments and selecting `Generator`, it hangs after submitting the `Enter Node versions (comma separated)` input

**Comment by ManfredLange** [comment]
Also hangs with Node v10.14.2 on Windows 10, 64 bit.

**Comment by bisubus** [comment]
The issue is still applicable.

**Comment by mshima** [comment]
Should be fixed with inquirer 7.0.2
